src/app/portlets/content-types/main/content-types.component.spec.ts

```diff
@@ -200,19 +203,11 @@ describe('ContentTypesPortletComponent', () => {
         }]);
     });
 
-    it('should have ONLY remove action because no publish environments are created', () => {
+    it('should have remove and add to bundle actions if is not community license and no publish environments are created', () => {
```

Comment:
Are you sure that add to bundle works without a license?

Reply:
Add to bundle don't work without a license, it should work?